hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -110,8 +110,9 @@ const unsigned REGISTRATION_TIMEOUT = 10 * 60 * 1000;
 using LacType = decltype(CellularGlobalIdentity::location_area_code);
 using CidType = decltype(CellularGlobalIdentity::cell_id);
 
-const system_tick_t UBLOX_NCP_R4_LARGE_PACKET_TIMEOUT_MS = 250;
-const size_t UBLOX_NCP_R4_LARGE_PACKET_THRESHOLD_SIZE = 512;
+const size_t UBLOX_NCP_R4_BYTES_PER_WINDOW_THRESHOLD = 512;
+const system_tick_t UBLOX_NCP_R4_WINDOW_SIZE_MS = 50;
```

Comment:
Tested that it could go down to `50ms` but cannot go higher than `512bytes`

Reply:
I have not tested lesser than `50ms` thinking that `50ms` is probably safer than something like `10ms`?